appveyor.yml

```diff
@@ -0,0 +1,270 @@
+  - '%CMD_IN_ENV% pip install -U --timeout 5 --retries 2 -r tools/ci/appveyor/requirements.txt'
+
+  - ps: |
+      $CurrentDir = $(get-location).Path;
+      $PatchFile="$CurrentDir/tools/ci/appveyor/numpy.patch"
+      $NumpyDir = (python -c 'import os; import numpy; print(os.path.dirname(numpy.__file__))') | Out-String
+      $NumpyDir = $NumpyDir.Trim()
+      cd "$NumpyDir"
+      cd ..
+      Get-Content $PatchFile | patch -p1 --merge
+      cd "$CurrentDir"
+
+build_script:
+  - ps: |
+      If ($env:PYTHON_ARCH -eq 32) {
+          $env:MINGW = $env:MINGW_32 
+      } Else {
+          $env:MINGW = $env:MINGW_64
+      }
+      $env:Path += ";$env:MINGW"
+      python setup.py build bdist_wheel
+
+      ls dist -r | Foreach-Object {
+          appveyor PushArtifact $_.FullName
+          pip install $_.FullName
+      }
+
+test_script:
+  - ps: |
+      $CurrentDir = $(get-location).Path;
+      $ScipyDir = (python -c 'import os; import sys; sys.path.remove(""""); import scipy; print(os.path.dirname(scipy.__file__))') | Out-String
+      $ScipyDir = $ScipyDir.Trim()
+      cp tools\ci\appveyor\setup.cfg $ScipyDir\..
+      cd "$ScipyDir\.."
+      py.test scipy -n6 --timeout=60 --junitxml=junit-results.xml
+      $ExitCode = $LastExitCode
+      echo $ExitCode
+      cp junit-results.xml $CurrentDir
+      cd "$CurrentDir"
+      if($ExitCode -ne 0) { $host.SetShouldExit($ExitCode )  }
+
+on_success:
+  - ps: |
+      If ($env:APPVEYOR_REPO_TAG -eq "true") {
+          git fetch --all
+          pip install twine
+          python setup.py sdist bdist_wheel bdist_wininst bdist_egg
+          Invoke-Expression "twine upload -u $env:PYPI_USERNAME -p $env:PYPI_PASSWORD --repository-url https://upload.pypi.org/legacy/ --skip-existing dist/*" 2>&1 | Out-Null
+      }
+
+
+after_build:
+  # Remove old or huge cache files to hopefully not exceed the 1GB cache limit.
+  #
+  # If the cache limit is reached, the cache will not be updated (of not even
+  # created in the first run). So this is a trade of between keeping the cache
+  # current and having a cache at all.
+  # NB: This is done only `on_success` since the cache in uploaded only on
+  # success anyway.
+  - C:\cygwin\bin\find "%LOCALAPPDATA%\pip" -type f -mtime +360 -delete
+  - C:\cygwin\bin\find "%LOCALAPPDATA%\pip" -type f -size +10M -delete
+  - C:\cygwin\bin\find "%LOCALAPPDATA%\pip" -empty -delete
+  # Show size of cache
+  - C:\cygwin\bin\du -hs "%LOCALAPPDATA%\pip\Cache"
+
+  - C:\cygwin\bin\find "C:\ProgramData\chocolatey" -type f -mtime +360 -delete
+  - C:\cygwin\bin\find "C:\ProgramData\chocolatey" -type f -size +10M -delete
+  - C:\cygwin\bin\find "C:\ProgramData\chocolatey" -empty -delete
+  # Show size of cache
+  - C:\cygwin\bin\du -hs "C:\ProgramData\chocolatey"
+
+
+on_finish:
+  - ps: |
+      (new-object net.webclient).UploadFile(
+        "https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)",
+        (Resolve-Path .\junit-results.xml)
```

Comment:
It works sluggishly on chrome and I opened appveyor/ci#1682 to address this issue on other browsers.